providers/os/connection/snapshot/volumemounter.go

```diff
@@ -107,8 +108,34 @@ func (m *VolumeMounter) mountVolume(fsInfo *fsInfo) error {
 	if fsInfo.FsType == "xfs" {
 		opts = append(opts, "nouuid")
 	}
```

Comment:
we'll probably be able to organize this logic a bit more once https://github.com/mondoohq/cnquery/pull/4101 is in as well. 
that could allow us to kinda "tier" the options...like this LVM check feels like it could  be a little earlier in the func?